identity/index.html

```diff
@@ -370,42 +368,9 @@ <h4><dfn>CaptureHandleChangeEvent</dfn></h4>
           <pre class="idl">
             [Exposed=Window]
             interface CaptureHandleChangeEvent : Event {
-              constructor(optional CaptureHandleChangeEventInit init = {});
-              [SameObject] CaptureHandle captureHandle();
+              constructor();
             };
           </pre>
-          <dl data-link-for="CaptureHandleChangeEvent" data-dfn-for="CaptureHandleChangeEvent">
-            <dt>
-              <dfn>captureHandle</dfn>
-            </dt>
-            <dd>
-              <p>
-                The track's {{CaptureHandle}} at the time the event was fired, as [=observable=] by
-                the capturing application. If not [=observable=] by the capturing application, all
-                of {{CaptureHandle}}'s fields will be set to their default value - the empty
-                {{DOMString}}.
-              </p>
-            </dd>
-          </dl>
-        </section>
-        <section>
-          <h4><dfn>CaptureHandleChangeEventInit</dfn></h4>
-          <pre class="idl">
-            dictionary CaptureHandleChangeEventInit : EventInit {
-              CaptureHandle captureHandle;
-            };
-          </pre>
-          <dl
-            data-link-for="CaptureHandleChangeEventInit"
-            data-dfn-for="CaptureHandleChangeEventInit"
-          >
-            <dt>
-              <dfn>captureHandle</dfn>
-            </dt>
-            <dd>
-              <p>The track's {{CaptureHandle}} at the time the event was fired.</p>
-            </dd>
-          </dl>
         </section>
         <section id="oncapturehandlechange">
           <h4><dfn>oncapturehandlechange</dfn></h4>
```

Comment:
`oncapturehandlechange` below should be defined as `Fired on a MediaStreamTrack when its observable CaptureHandle changes`.

Reply:
Should be fixed by #56.